src/lib/common/versionInfo.h

```diff
@@ -0,0 +1,42 @@
+#ifndef SRC_LIB_COMMON_VERSIONINFO_H_
+#define SRC_LIB_COMMON_VERSIONINFO_H_
+
+/*
+*
+* Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+*
+* This file is part of Orion Context Broker.
+*
+* Orion Context Broker is free software: you can redistribute it and/or
+* modify it under the terms of the GNU Affero General Public License as
+* published by the Free Software Foundation, either version 3 of the
+* License, or (at your option) any later version.
+*
+* Orion Context Broker is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+* General Public License for more details.
+*
+* You should have received a copy of the GNU Affero General Public License
+* along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+*
+* For those usages not covered by this license please contact with
+* iot_support at tid dot es
+*
+* Author: Burak Karaboga - ATOS Resarch and Innovation
+*/
+
+/* ****************************************************************************
+*
+* Release Date - The date of this version's release in YYYY-MM-DD format but other formats are also allowed.
+*/
+#define RELEASE_DATE  "Mon Sep 11 19:22:01 CEST 2017"
+
+/* ****************************************************************************
+*
+* API Documentation - The link to the the GEri documentation, either in the gh-pages (.github.io/) inside the fiware organization in GitHub or ReadTheDocs manual.
+*/
+#define API_DOC       "https://fiware-orion.readthedocs.org/en/master/"

```

Comment:
A better place for this would be lib/common/defaultValues.h

Reply:
Note also that as API_DOC and RELEASE_DATE will be moved out of this file, the whole versionInfo.h is unnecessary.

Reply:
Fixed in e88551b